chia/full_node/full_node_api.py

```diff
@@ -1147,16 +1132,16 @@ async def request_additions(self, request: wallet_protocol.RequestAdditions) ->
         if request.puzzle_hashes is None:
             for puzzle_hash, coins in puzzlehash_coins_map.items():
                 coins_map.append((puzzle_hash, coins))
-            response = wallet_protocol.RespondAdditions(block.height, block.header_hash, coins_map, None)
+            response = wallet_protocol.RespondAdditions(request.height, header_hash, coins_map, None)
         else:
+            coins_map_filtered = [(ph, c) for (ph, c) in coins_map if ph in request.puzzle_hashes]
```

Comment:
isn't `coins_map` always empty here?

Reply:
this should probably be called before `response` is generated?

Reply:
uhh yeah you're right... good catch. Should have added a test here.